UserGuide.md

```diff
@@ -506,17 +517,20 @@ The `--created_by email` flag is optional and defaults to the email address in y
 
 `TEMPLATE` is the ID or name of the template you wish to map.  If you need to find the ID of a specific template, you can get it using  `carrot_cli template find.`
 
-
 `REPORT` is the ID or name of the report you wish to map.  If you just completed step 2 and created a report, the ID of that report should have been included in the results from that command.  If not, you can find the report using `carrot_cli report find`.
 
 `REPORT_TRIGGER` is an optional argument that corresponds to the event that should trigger the generation of the report.  It defaults to `single`, which means a report will be generated upon successful completion of a single run of a test based on the template.  The other option is `pr`, which means a report will be generated upon successful completion of a [PR Comparison run](#starting-a-pr-comparison-run-from-github).
 
 
+#### <a name="generating-a-report"/> **Generating a Report**
 
+Filled reports can be generated in three ways. Once the report has been generated, it will be available in 3 forms: the Jupyter notebook that is produced by CARROT before being run, the filled Jupyter notebook that is created once all the cells are run, and a static HTML representation of the filled notebook.
```

Comment:
comment on where those get populated to

Reply:
Oh that's a good idea.  I should tell people where to get those